piqe_ocp_lib/tests/resources/test_ocp_control_planes.py

```diff
@@ -0,0 +1,43 @@
+import logging
+import pytest
+import random
+from piqe_ocp_lib.api.resources.ocp_control_planes import OcpControlPlane
+from piqe_ocp_lib import __loggername__
+
+logger = logging.getLogger(__loggername__)
+
+pytest.control_plane_components_names = None
+
+
+@pytest.fixture(scope="session")
+def ocp_control_plane(get_kubeconfig):
+    kube_config_file = get_kubeconfig
+    return OcpControlPlane(kube_config_file=kube_config_file)
+
+
+class TestOcpControlPlane:
```

Comment:
Please provide PyDoc for all test methods.

Reply:
Done